api/src/test/java/io/druid/data/input/impl/TimestampSpecTest.java

```diff
@@ -37,6 +39,29 @@ public void testExtractTimestamp() throws Exception
   }
 
   @Test
+  public void testExtractTimestampWithTimezone() throws Exception
+  {
+    final DateTimeZone saved = DateTimeZone.getDefault();
+    final DateTimeZone dataTimezone = DateTimeZone.forID("Asia/Shanghai");
+    try {
+      DateTimeZone.setDefault(DateTimeZone.UTC);
```

Comment:
@gaodayue we already force all tests to run with UTC timezone, so you don't need to set it here.